pkg/resource/endpoint/custom_update_conditions.go

```diff
@@ -17,48 +17,26 @@
 package endpoint
 
 import (
-	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
 	svcapitypes "github.com/aws-controllers-k8s/sagemaker-controller/apis/v1alpha1"
-	"github.com/aws/aws-sdk-go/aws"
+	customShared "github.com/aws-controllers-k8s/sagemaker-controller/pkg/common"
 	svcsdk "github.com/aws/aws-sdk-go/service/sagemaker"
-	corev1 "k8s.io/api/core/v1"
 )
 
+var terminalMessage = "endpoint status: Failed. Check FailureReason"
```

Comment:
Keep all the error messages in the same format

Reply:
Also usually variable names for constants are capitalized.  
Check the ack convention and follow the same

Reply:
or atleast use `svcsdk.EndpointStatusFailed` instead of Failed. Also, did we consider printing the failure reason here ?

Reply:
failure reason will be set in a separate field under status